components/story-based-tests.js

```diff
@@ -49,7 +49,7 @@ export ProgressIndicator from '../components/progress-indicator/__docs__/storybo
 // export Tabs from '../components/tabs/__docs__/storybook-stories';
 // export Textarea from '../components/textarea/__docs__/storybook-stories';
 // export Toast from '../components/toast/__docs__/storybook-stories';
-// export Tooltip from '../components/tooltip/__docs__/storybook-stories';
+export Tooltip from '../components/tooltip/__docs__/storybook-stories';
```

Comment:
@interactivellama, correct me if I'm wrong but I believe we are trying to move away from this file and use `storybook-stories.js`.

Is there a reason why you needed to uncomment this?

Reply:
I wasn't too clear on the best way to add a snapshot test for my change, this seemed like [the recommended approach](https://github.com/salesforce/design-system-react/blob/master/components/storybook-stories.js#L5-L7) but let me know if there is a newer way.

Reply:
Adding a story to the Storybook dev environment and then uncommenting here is the current best way to test classname additions.

We need to go through and uncomment most of these now that Jest is running with JSDOM instead of Node.